core/templates/dev/head/i18n.js

```diff
@@ -75,8 +75,9 @@ oppia.config([
   var availableLanguageKeys = [];
   var availableLanguageKeysMap = {};
   for (var prop in GLOBALS.SUPPORTED_SITE_LANGUAGES) {
-    availableLanguageKeys.push(prop);
-    availableLanguageKeysMap[prop + '*'] = prop;
+    availableLanguageKeys.push(GLOBALS.SUPPORTED_SITE_LANGUAGES[prop].id);
```

Comment:
In output of presubmit checks a total of 457 of 531 tests fail, so the output is very long

I just inserted 

```
GLOBALS.SUPPORTED_SITE_LANGUAGES.forEach(function(language) {
    console.log(language);
  });
```
 this piece of code in i18n.js (above the previous loop) like this:

```
var availableLanguageKeys = [];
  var availableLanguageKeysMap = {};
  GLOBALS.SUPPORTED_SITE_LANGUAGES.forEach(function(language) {
    console.log(language);
  });
  for (var prop in GLOBALS.SUPPORTED_SITE_LANGUAGES) {
    availableLanguageKeys.push(GLOBALS.SUPPORTED_SITE_LANGUAGES[prop].id);
    availableLanguageKeysMap[GLOBALS.SUPPORTED_SITE_LANGUAGES[prop].id + '*'] =
    GLOBALS.SUPPORTED_SITE_LANGUAGES[prop].id;
  }
  availableLanguageKeysMap['*'] = 'en';
```

which causes the above error. If I remove the above loop, all the tests pass!

Going through the errors the cause of each error is:

```
Error: [$injector:modulerr] Failed to instantiate module oppia due to:
	TypeError: Cannot read property 'forEach' of undefined
```